### PR TITLE
s/Gitter/Zulip/g

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Hail
 
-[![Gitter](https://badges.gitter.im/hail-is/hail.svg)](https://gitter.im/hail-is/hail?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![CI Status](https://ci.hail.is/app/rest/builds/buildType:(id:HailSourceCode_HailCi)/statusIcon)](https://ci.hail.is/viewType.html?buildTypeId=HailSourceCode_HailCi&branch_HailSourceCode_HailMainline=%3Cdefault%3E&tab=buildTypeStatusDiv)
+[![](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://hail.zulipchat.com) [![CI Status](https://ci.hail.is/app/rest/builds/buildType:(id:HailSourceCode_HailCi)/statusIcon)](https://ci.hail.is/viewType.html?buildTypeId=HailSourceCode_HailCi&branch_HailSourceCode_HailMainline=%3Cdefault%3E&tab=buildTypeStatusDiv)
```
Comment (jigold) [inline README.md]: This isn't removing the zulip badge on the docs preview.

File: python/hail/docs/_templates/layout.html [hunk 2]
```diff
@@ -129,13 +129,7 @@
...
           </li>
             <li id="forum" class="nav-item"><a href="http://discuss.hail.is">FORUM</a></li>
-            <li id="chat" class="nav-item dropdown">
-            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">CHAT<span class="caret"></span></a>
-            <ul class="dropdown-menu dropdown-menu-left">
-              <li><a class="dropdown-item" href="https://gitter.im/hail-is/hail">General</a></li>
-              <li><a class="dropdown-item" href="https://gitter.im/hail-is/hail-dev">Developers</a></li>
-            </ul>
-          </li>
+            <li id="chat" class="nav-item"><a href="http://hail.zulipchat.com">CHAT</a></li>
```
Comment (jigold) [inline python/hail/docs/_templates/layout.html]: Unfortunately, I couldn't test the link on the docs preview. Got an error message relating to security. Can you test it works locally?

File: README.md (hunk 1 above)
Comment (tpoterba) [reply README.md]: fixed.

File: python/hail/docs/_templates/layout.html (hunk 2 above)
Comment (tpoterba) [reply python/hail/docs/_templates/layout.html]: it works!

**Pull request merged**